tests/run/cpp_stl_string.pyx

```diff
@@ -102,6 +102,15 @@ def test_push_back(char *a):
     s.push_back(<char>ord('s'))
     return s.c_str()
 
+def test_pop_back(char *a):
+    """
+    >>> test_pop_back(b'abc')
+    b'ab'
```

Comment:
This doesn't work in Py2. I sometimes write test like this:
```suggestion
    >>> test_pop_back(b'abc') == b'ab'  or  test_pop_back(b'abc')
    True
```
(Technically, the test would then still succeed if the function call returned `True`, but then, let's assume that we're all honest developers who wouldn't let that pass through a code review…)

Reply:
Ah, ok. I wrote it that way, because it produces a useful error message if the test fails, which `assert True` does not.

Reply:
Fixed.

Reply:
I know, that's exactly why I use the spelling above. If the comparison fails, I can see the actual output.

Reply:
Hmm. Makes sense.